docs/how-to/construct-models.md

```diff
@@ -382,25 +382,98 @@ ans =
 
 ### Writing custom plot functions that can be manipulated
 
-
 # Creating new conductances
 
-## Understanding the `conductance` class
+> code testing this functionality exists in `../xolotl/examples/demo_conductance`.
+
+The `conductance` class exists to generate custom `C++` header files for novel
+conductances without leaving the `MATLAB` prompt.
+
+First, instantiate a `conductance` object. Remember that you can always see all the
+properties of the object by typing its name in to the command window, or
+`fieldnames(temp)`.
+
+```matlab
+temp = conductance;
```

Comment:
don't call variables temp

Reply:
instead, the variable should be the name of the channel you want to create (e.g., NaV)